docs/antora/modules/ROOT/pages/management/back-ups.adoc

```diff
@@ -0,0 +1,32 @@
+= Applying DB Snapshots
+Chris Braithwaite <christopher.braithwaite@lightcurve.io>
+:description: Describes how to perform Lisk Service back-ups
+:toc:
+:idseparator: -
+:idprefix:
+:experimental:
+:imagesdir: ../assets/images
+:source-language: bash
+:url_snapshot_config: configuration/index.adoc
+
+If you wish to apply database snapshots for both your Docker and Source-based installation, which are very similar this is covered in the following sections below.
+
+
+== Applying snapshots using Docker
+
+With regard to applying snapshots for the Docker installation, the configurations to sync Lisk Service from an existing snapshot can be used to speed up the sync process.
+It is possible to enable or create a snapshot of the MySQL database, and the available xref:{url_snapshot_config}[MySQL snapshot configurations] can be found in the Lisk Service configuration reference page.
+To change the configuration setting or to create either a MariaDB or a MySQL snapshot, set the `ENABLE_APPLY_SNAPSHOT` to `true`.
+
+To enable the MySQL snapshot to be imported automatically, please ensure to set the aforementioned parameters above before starting the service.
+//TODO: Once the new Add backups page for Lisk Service is completed, add the following: If you wish to import the database manually, please refer to the xref:{add_backups.adoc}[Import Snapshot] section.
```

Comment:
Which page are we talking about here?

Reply:
This is not applicable now, so the TODO has been removed.